src/api/index.js

```diff
@@ -0,0 +1,20 @@
+import axios from 'axios'
+
+export default class {
+  constructor(client) {
```

Comment:
I think we should not write code in `index.js` files. Only imports/exports and some simple logic. However, then you have to think about class names :)